osf/metadata/serializers/datacite/datacite_tree_walker.py

```diff
@@ -388,6 +392,20 @@ def _visit_related(self, parent_el):
                 datacite_relation,
             )
 
+        # Then add verified links to same relatedIdentifiers element
+        osf_item = self.basket.focus.dbmodel
+        verified_links = getattr(osf_item, 'verified_resource_links', None)
+        if verified_links:
+            for link, resource_type in verified_links.items():
+                if link and isinstance(link, str) and smells_like_iri(link):
```

Comment:
I guess this check is here to make sure link looks like a link, which is good.

* Is `smells_like_iri()` precise enough? (i.e. not too loose or too strict)
* If so, it already checks the `str` type so we can skip it.